syft/frameworks/crypten/hook/hook.py

```diff
@@ -5,29 +5,68 @@
 import crypten
 import torch as th
 
-methods_to_hook = ["load"]
+
+class CrypTenPlanBuild(object):
+    @staticmethod
+    def f_return_none(*args, **kwargs):
+        return None
+
+    @staticmethod
+    def f_return_cryptensor(*args, **kwargs):
+        return crypten.cryptensor(th.zeros([]))
+
+    @staticmethod
+    def f_return_model_or_cryptensor(*args, **kwargs):
+        """
+        # TODO: When we solve the problem converting OnnxModels to PyTorch we can
+        have serialized models on workers that we can load without having the party
+        that started the computation know about the instrinsics of the architecture
+        if args[0] == "crypten_model":
+            return crypten.nn.Module()
+        else:
+        """
+        return crypten.cryptensor(th.zeros([]))
+
+
+crypten_to_auto_overload = {
+    crypten.mpc.MPCTensor: ["get_plain_text"],
+    crypten.nn.Module: ["encrypt", "decrypt", "__call__"],
+}
+
+crypten_plan_hook = {
```

Comment:
Could you add a short comment to explain what are the difference between `crypten_to_auto_overload` and `crypten_plan_hook`.
If the role of `crypten_plan_hook` looks quite obvious, the one of `crypten_to_auto_overload` is not as clear at first sight.

In particular, 
- Why get_plain_text appears only in the first dict?
- What would happen if `crypten_to_auto_overload` whasn't there?

Reply:
Done

Reply:
Done